dashboard/src/components/Config/AppRepoList/AppRepoForm.tsx

```diff
@@ -219,6 +246,18 @@ export function AppRepoForm(props: IAppRepoFormProps) {
     setSkipTLS(!skipTLS);
     setValidated(undefined);
   };
+  const handleFilterNames = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setFilterNames(e.target.value);
+    setValidated(undefined);
```

Comment:
Out of curiosity, why do we need these `setValdiated` ? Which validation is being performed in these cases?

Reply:
Right, copy-paste error here, we are not validating anything regarding the filter.